### PR TITLE
Avoid all the “Android” OS reports 

### PR DESCRIPTION
**Pull request by andydotxyz**
As discussed on slack

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -62,27 +62,11 @@ body:
...
-  validations:
-    required: true
 - type: input
   attributes:
-    label: Operating system version
-    description: Clarify the version of the operating system you are using.
-    placeholder: "e.g. Windows 10, 10.13 High Sierra or Ubuntu 18.04"
+    label: Operating system and version
+    description: The name and version of the operating system you are using.
+    placeholder: "e.g. Windows 10, macOS High Sierra or Ubuntu 18.04"
```
Comment (Jacalz) [inline .github/ISSUE_TEMPLATE/bug_report.yml:69]: While you're at it, would you mind updating the Ubuntu reference to something newer? Version 18.04 has Go 1.10 (and 20.04 seems to have Go 1.13) as the default repo version 😅

**Review by Jacalz — APPROVED** [review]
Looks good. I left a bit on a possible improvement while you're at it but will approve now as it isn't super important :)

**Comment by Jacalz** [comment]
You might also want to merge this into master unless you plan to cherry-pick it over :)

**Comment by andydotxyz** [comment]
Thanks, will pick over to master. I did not get round to version name updates, it's just examples :)

**Pull request merged**